### PR TITLE
#165377507 Update READ.me to Include setup Instructions

### PR DESCRIPTION
**Pull request by fejiroofficial**
#### What does this PR do?
- updates the readme file to include a description for the project set up

#### Description of Task to be completed?
- add the description for the set up to the repository

#### What are the relevant pivotal tracker stories?
[#165377507](https://www.pivotaltracker.com/story/show/165377507)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,71 @@
...
+```
+
+> Environment variables (Provide details only for the empty data slots)
+
+```
+# Cloudinary cloud name
+CLOUDINARY_CLOUD_NAME=
+
+# Facebook auth url
+FACEBOOK_AUTH_URL=
```
Comment (aanchirinah) [inline README.md:68]: #### Nitpick
- We can add a `.env.example` reference to this

File: README.md [hunk 2]
```diff
@@ -2,9 +2,71 @@
...
+# Install package modules
+npm install
+
+# To run test
+npm run test
+
+# Start up the server
+npm run start:dev
+
+```
```
Comment (aanchirinah) [inline README.md:59]: #### Nitpick
- List the features of the application and indicate those that work now.

**Review by aanchirinah — CHANGES_REQUESTED** [review]
Good job @fejiroofficial , kindly attend to my requested changes.

**Comment by thislekan** [comment]
Is this PR ready or are we still implementing feedback? @fejiroofficial

File: README.md [hunk 3]
```diff
@@ -2,9 +2,111 @@
...
+# Web api base url
+API_BASE_URL=
+```
+
+> Examples can be found in .env.example
+
+```
+CLOUDINARY_CLOUD_NAME=erty51cvb
+APP_KEY=secret
+FACEBOOK_AUTH_URL=http://localhost:3000/api/v1/auth/facebook
```
Comment (aanchirinah) [inline README.md]: #### Nitpick
```suggestion
FACEBOOK_AUTH_URL=http://localhost:<PORT_NUMBER>/api/v1/auth/facebook
```

**Review by aanchirinah — CHANGES_REQUESTED** [review]
Nice job but review my changes.

**Review by valenstical — APPROVED** [review]
LGTM

**Review by davdwhyte87 — APPROVED** [review]
LGTM

**Review by devdbrandy — APPROVED** [review]
Looks good to me.

**Pull request closed without merging**